src/main/resources/db/data/V1_37__add_staff_for_no_tier.sql

```diff
@@ -0,0 +1,11 @@
+Insert into TEAM (TEAM_ID, CODE, DESCRIPTION, DISTRICT_ID, LOCAL_DELIVERY_UNIT_ID, TELEPHONE, UNPAID_WORK_TEAM, ROW_VERSION,
```

Comment:
Is this insert necessary? The tests don't seem to refer to this

Reply:
They are needed to create the contact https://github.com/ministryofjustice/community-api/blob/main/src/main/java/uk/gov/justice/digital/delius/service/TierService.java#L56...L57

Reply:
Is it because you are using this offender in a tiering test for the first time? X330899
If so makes sense

Reply:
This happened when adding the no tier exists test, X330899 has got a tier already so used X320741 as it doesn't have an existing tier